lib/commands/admin/enableCommand.js

```diff
@@ -0,0 +1,27 @@
+const source = require('rfr');
+
+const { sendMessage } = source('lib/utils/util');
+const { Settings } = source('lib/database/models/settings');
+
+function addMemeSubCommand(message, args) {
```

Comment:
Change name of function to enableCommand

Reply:
ooops copy paste